optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/test/resources/META-INF/beans.xml

```diff
@@ -14,12 +14,11 @@
   ~ limitations under the License.
   -->
 
-<beans>
-  <scan>
-    <exclude name="**.client.**"/>
```

Comment:
I see. But since the above mentioned `beans.xml` is in `src/test/resources` it should not influence what ends up in WEB-INF/lib. It is used just for testing (as those alternatives can not be used in th real app). It is true that It could be slower if the client classes are being scanned.